### ISSUE TITLE
Response to CVE-2024-26455?

### ISSUE THREAD

**Issue by risicle** [fluent/fluent-bit#8539]
I noticed CVE-2024-26455 has been published but I see no mention of it here - it's possible you haven't been properly informed by the reporter. Either way, I thought it best to have a public place to discuss/track the issue (if it even is an issue) seeing as it's already public.

**Comment by LuMingYinDetect** [comment]
Hello! I am the discoverer of this CVE. First of all, I apologize for not contacting you in advance because this is my first time applying for a CVE, and I may not be familiar enough with the application process. This time, I found two issues in the software in total, and I submitted an issue two days ago. Here are our discussion links: https://github.com/fluent/fluent-bit/issues/8531%E3%80%81https://github.com/fluent/fluent-bit/issues/8530. The one you mentioned seems to be an external library according to my discussion with the developers. The other defect appears to be a double-free defect in the software. We are considering how to fix it, and I have already submitted a PR for your reference!

**Comment by LuMingYinDetect** [comment]
Hello! I am the discoverer of this CVE. First of all, I apologize for not contacting you in advance because this is my first time applying for a CVE, and I may not be familiar enough with the application process. This time, I found two issues in the software in total, and I submitted an issue two days ago. Here are our discussion links: https://github.com/fluent/fluent-bit/issues/8531 and https://github.com/fluent/fluent-bit/issues/8530. The one you mentioned seems to be an external library according to my discussion with the developers. The other defect appears to be a double-free defect in the software. We are considering how to fix it, and I have already submitted a PR for your reference!
